ui_framework/components/tabs/_tabs.scss

```diff
@@ -14,33 +18,42 @@
     color: $subduedFontColor;
     background-color: #ffffff; /* 1 */
     border: 1px solid $standoutBackgroundColor;
-    border-bottom: none;
+    border-bottom-width: 2px;
     border-radius: 0; /* 1 */
+    margin-bottom: -2px; /* 3 */
 
     & + & {
       border-left: none;
+
+      &:focus:not(.kuiTab-isSelected):not(:active) {
+        margin-left: -1px; /* 2 */
+      }
+    }
+
+    &:active {
+      outline: none !important; /* 1 */
+      box-shadow: none; /* 1 */
     }
 
     &:focus {
-      color: $linkColor;
+      outline: none; /* 1 */
+    }
+
+    &:focus:not(.kuiTab-isSelected):not(:active) {
       z-index: 1;
-      outline: 1px solid $selectedBorderColor !important; /* 1 */
+      color: $linkColor;
+      border: 1px solid $selectedBorderColor !important;
+      padding-bottom: 11px; /* 4 */
     }
 
     &:hover:not(.kuiTab-isSelected) {
       color: $linkHoverColor;
-    }
-
-    &:active {
-      outline: none;
-      border-color: #ffffff;
-      box-shadow: none; /* 1 */
-      color: $linkColor !important;
+      background-color: #F2F2F2;
```

Comment:
We have a set of "brand" grays I think... at least we used to. Is there a reason that you're using specific colors here (`#ffffff`as well) as opposed to variables?